### PR TITLE
Refactor into installable package

### PR DESCRIPTION
**Pull request by KevinMGranger**

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -0,0 +1,9 @@
+autobahn==0.18.2
+beautifulsoup4==4.5.3
+bs4==0.0.1
+-e git+git@github.com:sopython/rabbit.git@99cf02db5a983b7718712c194e39d9fdd55ec0c1#egg=rabbit
```
Comment (KevinMGranger) [inline requirements.txt]: Uh, not sure if this is supposed to be here. I've never actually used `requirements.txt` before.

**Comment by kms70847** [comment]
Ok, I'm trying this out... How do I run it? After I `git pull` and `pip install -e` it, running main.py just gives me:

```
c:\Programming\Github\rabbit\rabbit>py -3 main.py
Traceback (most recent call last):
  File "main.py", line 14, in <module>
    from .dbmodel import User, get_session
SystemError: Parent module '' not loaded, cannot perform relative import
```

**Comment by kms70847** [comment]
Andras suggested in chat that I do `py -3 -m rabbit.main`, and that takes care of the exception, but then nothing happens because `main()` doesn't get called anywhere. Adding a `main()` call causes the bot to start up as expected. I see PR #22 provides formal entry points in its setup.py... I'll accept this PR in the expectation that the PR following it resolves the small issue I'm having.

**Comment by adeak** [comment]
Actually, the first one was vaultah. I suggested adding a magical `main()` call :)

**Pull request merged**